assets/fma-template.md

```diff
@@ -42,60 +42,64 @@
 > - Include both implementation and operational failure modes
 > - Provide references to support the reviewer.
 > - The size of the document will likely be proportional to the project's complexity.
-> - The ultimate goal of this document is to identify action items to improve the security of the  project. The FMA review process can be accelerated by proactively identifying action items during the writing process.
+> - The ultimate goal of this document is to identify action items to improve the security of the project. The FMA review process can be accelerated by proactively identifying action items during the writing process.
 
 ## Introduction
 
-This document covers *[project name, high-level summary of the project, and scope of this analysis].*
+This document covers _[project name, high-level summary of the project, and scope of this analysis]._
 
 Below are references for this project:
 
-- *Link 1, e.g. project charter or design doc*
-- *Link 2, etc.*
+- _Link 1, e.g. project charter or design doc_
+- _Link 2, etc._
 
 ## Failure Modes and Recovery Paths
 
-***Use one sub-header per failure mode, so the full set of failure modes is easily scannable from the table of contents.***
+**_Use one sub-header per failure mode, so the full set of failure modes is easily scannable from the table of contents._**
 
-### [Name of Failure Mode 1]
+### FM1: [Name of Failure Mode 1]
```

Comment:
added a standard prefix to each failure mode to make the doc more readable, as headings are less distinctive in GH markdown vs. Notion.